fem/fespace.cpp

```diff
@@ -1633,7 +1851,9 @@ void FiniteElementSpace::Construct()
    MFEM_VERIFY(!NURBSext, "internal error");
 
    elem_dof = NULL;
+   elem_fos = NULL;
```

Comment:
What I mean is that the Space used to be a very cheap object originally, but we keep adding per-element data (like `elem_dof`), which I'm not sure is always good...

Reply:
(of course it's still better than per-DOF data, like those two `dof_*_array` members)

Reply:
Right, well like `elem_dof` this is just a convenience where we trade-off memory for cleaner code, I suppose.  It's not clear there would be a significant difference in speed. We have a few options. We could call `mesh::GetElementFaces` each time we need this information but this method returns more information than we need (i.e. the face indices themselves) so that's not optimal.  We could also restrict things so that this object is built only for high order Nedelec spaces or perhaps restrict even further based on the presence of triangular faces.

Reply:
OK, maybe it's cleaner to keep it.